corehq/toggles.py

```diff
@@ -849,6 +849,13 @@ def _commtrackify(domain_name, toggle_is_enabled):
     [NAMESPACE_DOMAIN],
 )
 
+USE_LIVE_QUERY = StaticToggle(
```

Comment:
Can you call this `FORMPLAYER_LIVE_QUERY` or something? And maybe put it below the other `LIVEQUERY` flag? This will be confusing otherwise.